src/content/platform-integration/web/faq.md

```diff
@@ -112,25 +112,6 @@ See [Embedding Flutter web][].
 
 See [Web content in Flutter][].
 
-### How do I debug a web app?
```

Comment:
Is this advice just wrong now?

Reply:
It's still correct, I just moved it to the "building" page for visibility.